courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<p>great names! I'd go all the way:</p><ul><li>targ_lang -&gt; target_language</li><li>inp_lang -&gt; input_language</li><li>inp_lang_tokenizer -&gt; input_language_tokenizer</li><li>etc.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>